server/pypi/packages/depthai/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: depthai
+  version: "2.24.0.0" # If a release is picked
+  # version: "2.24.0.0.dev0+7b57b28305368582d004d5c6ec2cffb66562f2e0"
+
+source:
+  git_url: https://github.com/luxonis/depthai-python.git
+  git_rev: 7b57b28305368582d004d5c6ec2cffb66562f2e0
+  # path: [path/to/depthai-python]
```

Comment:
I don't understand what "If a release is picked" means.

Also, if the commented-out `version` and `path` lines serve some purpose, please add more comments to explain what that is, or otherwise remove them.